middleware/grpc/shared.go

```diff
@@ -27,6 +27,24 @@ import (
 	"github.com/openzipkin/zipkin-go/model"
 )
 
+type handleRPCParser struct {
+	inPayload  func(*stats.InPayload, zipkin.SpanCustomizer)
```

Comment:
This functions might become `func(*stats.InPayload, model.SpanContext, zipkin.SpanCustomizer)` because `SpanCustomizer` does not expose access to the context.

Reply:
I would expect RPC parser to be slightly more abstract.. It is a bit easier though possibly easier to see this later, when there's a request and response type instead of matching exact grpc lifecycle. I mean it would be easier to see if another RPC model was in use. ex https://github.com/openzipkin/brave/blob/master/instrumentation/grpc/src/main/java/brave/grpc/GrpcRequest.java